wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -48,49 +76,49 @@ The [Nomination Assessment Team group page](https://osu.ppy.sh/groups/7) lists a
 
 ### osu!
 
-| Name | Additional languages | Primary responsibilities |
+| Name | Additional languages | Assigned subgroup |
```

Comment:
Maybe a TODO for the future, but might be a good idea to briefly describe what each of the advisors are actually doing, since it's a very broard term.

Reply:
yeah describing what the advisors do or have done as examples is much better than just stating "advisor" for sure

think the benefit of giving some examples far outweighs the detriment of people misinterpreting it as all they do and will do kinda thing, which im guessing uber was worried about here judging from the inital post

Reply:
it's that + i wasn't sure how to format this idea properly so i stick to the simplest solution, tho i definitely would like it to be more descriptive

Reply:
could either:
- do `Advisor (insert main task)`
- use footnotes to add details

![image](https://user-images.githubusercontent.com/62819481/202457151-c71230a9-6c63-487f-946f-63f992eaadcb.png)

Reply:
after some discussion we opted for footnotes